test/utils/Deployers.sol

```diff
@@ -80,8 +80,8 @@ contract Deployers {
     PoolKey uninitializedNativeKey;
 
     // Update this value when you add a new hook flag.
-    uint256 hookPermissionCount = 14;
-    uint160 clearAllHookPermisssionsMask = ~uint160(0) >> (hookPermissionCount);
+    uint160 hookPermissionCount = 14;
+    uint160 clearAllHookPermissionsMask = uint160(~((1 << hookPermissionCount) - 1));
```

Comment:
what youve got here is all 0s - it doesnt just clear the hook permissions it clears everything. I think instead it would be better as `~uint160(0) << hookPermissionCount` to have all 1s except the 14 lowest bits